#
# After failed patter match returning content of html request can break
# monasca-persister/influxdb. This might be unreported bug as it behaves
# in a same way in HEAD
#
--- a/collector/checks_d/http_check.py  	2017-05-09 08:30:30.067010000 +0000
+++ b/collector/checks_d/http_check.py	        2017-05-09 08:54:11.723010000 +0000
@@ -186,7 +186,7 @@
             if re.search(pattern, result_string, re.DOTALL):
                 self.log.debug("Pattern match successful")
             else:
-                error_string = 'Pattern match failed! "{0}" not in "{1}"'.format(pattern, result_string)
+                error_string = 'Pattern match failed! "{0}" not found on "{1}"'.format(pattern, addr)
                 self.log.info(error_string)
                 # maximum length of value_meta including {'error':''} is 2048
                 # Cutting it down to 1024 here so we don't clutter the